### ISSUE TITLE
The Results between testing and validation phase is inconsistent.

### ISSUE THREAD

**Issue by StrivedTye** [Ghostish/Open3DSOT#43]
Hi Authors,

Thanks for your excellent work.
I have a question when running your code. During the training phase, the validation set is equal to the testing set. But during the testing phase, when using "--test" to evaluate the trained network, the performances (succ./prec.)  are inconsistent with the validation's performance. Maybe not the reason of "model.eval()". Have you noticed this problem? Thanks.

**Comment by Ghostish** [comment]
This appears to be a bug related to DDP (Distributed Data Parallel) in PyTorch Lightning. I have noticed that this issue arises when the test sequences cannot be evenly distributed across all GPUs, such as in the case of having 11 sequences and 2 GPUs. To ensure accurate performance measurement, I recommend using the validation metrics as a reference and conducting the model testing on a **single** GPU.
